packages/circle-demo-webapp/app/routes/wallets.$id/route.tsx

```diff
@@ -16,25 +17,29 @@ export async function loader({ params }: { params: { id: string } }) {
     throw new Error('Wallet Set ID is required');
   }
 
-  const [wallets, walletSet] = await Promise.all([
-    sdk.wallet.list({ walletSetId: id }),
-    sdk.walletSet.get(id),
+  const [walletsResp, walletSetResp] = await Promise.all([
+    sdk.listWallets({ walletSetId: id }),
+    sdk.getWalletSet({ id }),
   ]);
 
   return {
-    wallets,
-    walletSet,
+    wallets: walletsResp?.data?.wallets,
+    walletSet: walletSetResp?.data?.walletSet,
```

Comment:
Probably casting here instead of casting inside the component?
```suggestion
    wallets: walletsResp?.data?.wallets as Wallet[],
    walletSet: walletSetResp?.data?.walletSet as WalletSet,
```